### PR TITLE
SSL mongo: certs parametrs

### PR DESCRIPTION
**Pull request by incognito1990**
##### SUMMARY
- add ssl parametrs for mongo template

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by incognito1990** [comment]
##### SUMMARY
- add ssl parametrs for mongo template

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME


##### ADDITIONAL INFORMATION

File: lib/ansible/modules/database/mongodb/mongodb_user.py [hunk 1]
```diff
@@ -341,6 +341,9 @@ def main():
             name=dict(required=True, aliases=['user']),
             password=dict(aliases=['pass'], no_log=True),
             ssl=dict(default=False, type='bool'),
+            ssl_certfile=dict(default=None),
```
Comment (dagwieers) [inline lib/ansible/modules/database/mongodb/mongodb_user.py:344]:
```suggestion
            ssl_certfile=dict(type='path'),
```

File: lib/ansible/modules/database/mongodb/mongodb_user.py [hunk 2]
```diff
@@ -341,6 +341,9 @@ def main():
...
+            ssl_keyfile=dict(default=None),
```
Comment (dagwieers) [inline lib/ansible/modules/database/mongodb/mongodb_user.py:345]:
```suggestion
            ssl_keyfile=dict(type='path'),
```

File: lib/ansible/modules/database/mongodb/mongodb_user.py [hunk 3]
```diff
@@ -341,6 +341,9 @@ def main():
...
+            ssl_ca_certs=dict(default=None),
```
Comment (dagwieers) [inline lib/ansible/modules/database/mongodb/mongodb_user.py:346]:
```suggestion
            ssl_ca_certs=dict(type='path'),
```

**Review by dagwieers — CHANGES_REQUESTED** [review]
A few remarks:
- I would make this behave differently, enable ssl automatically when ssl-parameters are specified.
- I was expecting dependencies between ssl-options. Are all these SSL options independent ?
- These options should go in all MongoDB modules, and we should have a mongodb module_utils and doc_fragments (currently this PR has them undocumented)

**Comment by dagwieers** [comment]
@mikhail-yarosh @ymrsmns It appears you want/need this functionality, it helps if people review PRs to get them merged. Aside from other users (i.e. the community) we shouldn't expect a higher entity to accept and merge new functionality.

If you like to help out, [join the MongoDB Working Group](https://github.com/ansible/community/wiki/MongoDB).

**Comment by mikhail-yarosh** [comment]
@dagwieers ok, understood... so, should we report an issue or feature request or whatever to see this functionality in future?

**Pull request closed without merging**